packages/react-components/src/store/index.ts

```diff
@@ -0,0 +1,12 @@
+import { create } from 'zustand';
+import { devtools } from 'zustand/middleware';
+import { createTrendCursorsSlice, TrendCursorsState } from './trendCusorSlice';
+
+export type StateData = TrendCursorsState;
+const useDataStore = create<StateData>()(
+  devtools((...args) => ({
+    ...createTrendCursorsSlice(...args),
+  }))
+);
+
+export default useDataStore;
```

Comment:
The dev tools need be passed information about trend cursors?

Reply:
not sure if i understand the question..... if you asking why devtools ? i added it for debugging purpose... https://github.com/pmndrs/zustand#typescript-usage